google-cloud-bigquerystorage/src/test/proto/jsonTest.proto

```diff
@@ -152,8 +152,9 @@ message TestTimestamp {
   optional int64 test_int = 4;
   optional int64 test_float = 5;
   optional int64 test_offset = 6;
-  optional int64 test_timezone = 7;
-  optional int64 test_saformat = 8;
+  optional int64 test_zero_offset = 7;
```

Comment:
To confirm: This was added to help with the offset test, right?

Reply:
Yes, it's meant to confirm that valid offsets such as `+12:34` or "no offset" strings such as `Z` and `+00:00` are treated correctly.